### PR TITLE
Image list

### PR DESCRIPTION
**Pull request by CTGitWhiz**
Please check! THX

US1 --> https://github.com/users/CTGitWhiz/projects/1/views/1?pane=issue&itemId=22167676

### CODE REVIEW

File: pages/components/ImageList.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  const images = Array.from(
+    { length: 20 },
+    (_, i) => `/../public/Images/image${i + 1}.png`
+  );
+
+  return (
+    <ImageListWrapper>
+      {images.map((src, i) => {
+        return (
+          <ImageWrapper key={i}>
```
Comment (Kheirah) [inline pages/components/ImageList.js]: The index is not enough for the `key`prop. You need a stable id.

File: pages/components/ImageList.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  margin: 20px;
+`;
+
+const ImageList = () => {
+  const images = Array.from(
+    { length: 20 },
+    (_, i) => `/../public/Images/image${i + 1}.png`
```
Comment (Kheirah) [inline pages/components/ImageList.js]: You need to generate a url that is accessible from the web and not just from within your project. Currently, the images aren't visible on Vercel's [deployment](https://capstone-project-2f43cc04n-ctgitwhiz.vercel.app/).

**Review by Kheirah — CHANGES_REQUESTED** [review]
Very good!

Have a look at the path of your images and figure out how to make your images visible on deployments as well.

**Comment by Kheirah** [comment]
Next time, please leave a link to your user story in the description of your pull request. And be more descriptive of the work you've done on that PR.

File: pages/components/ImageList.js (hunk 1 above)
Comment (CTGitWhiz) [reply pages/components/ImageList.js]: ? stable id --> 

 const id = uid(); // generate a unique id
        return (
          <ImageWrapper key={id}>

File: pages/components/ImageList.js (hunk 2 above)
Comment (CTGitWhiz) [reply pages/components/ImageList.js]: I thought this is the way to start? Since i should implement something like "cloudinary" at the end...?

Comment (Kheirah) [inline pages/components/ImageList.js]:
```suggestion
    (_, i) => `/Images/image${i + 1}.png`
```
Maybe this helps 😄

Comment (CTGitWhiz) [reply pages/components/ImageList.js]: fixed it

**Comment by CTGitWhiz** [comment]
I think, I fixed all issues so far..

**Comment by TuncAlt** [comment]
Looks good!

**Review by TuncAlt — APPROVED** [review]

File: pages/components/ImageList.js (hunk 2 above)
Comment (Kheirah) [reply pages/components/ImageList.js]: There **is** a difference. Images are now visible on Vercel through this change.

**Pull request merged**